### ISSUE TITLE
[x64] Assertion failed 'cls != NO_CLASS_HANDLE'

### ISSUE THREAD

**Issue by VincentBu** [dotnet/coreclr#23197]
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstress1_tst/178/

**log:**
```
Assert failure(PID 68338 [0x00010af2], Thread: 68338 [0x10af2]): Assertion failed 'cls != NO_CLASS_HANDLE' in 'JIT.HardwareIntrinsics.X86.SimpleUnaryOpTest__ConvertToVector128Int16Byte:RunClsVarScenario():this' (IL size 56)
```

**Comment by BruceForstall** [comment]
I also see:
```
Assert failure(PID 49227 [0x0000c04b], Thread: 49289 [0xc089]): Assertion failed 'cls != NO_CLASS_HANDLE' in 'System.Net.WebSockets.ManagedWebSocket:ApplyMask(struct,int,int):int' (IL size 254)
```
here:
https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_ubuntu_corefx_jitstress1/883/consoleFull

**Comment by CarolEidt** [comment]
Fixed with #23193
